src/main/java/rssreader/operations/DbOperation.java

```diff
@@ -0,0 +1,17 @@
+package rssreader.operations;
+
+import org.hibernate.Session;
+import rssreader.util.HibernateUtil;
+
+public abstract class DbOperation {
+    public void execute() {
+        try(Session session = HibernateUtil.getSessionFactory().openSession())
+        {
```

Comment:
Strange formatting: I mean that normally you don't start new line with '{', but in db operation classs, you do